dev/NavigationView/NavigationView.xaml

```diff
@@ -470,7 +472,7 @@
                                 OpenPaneLength="{TemplateBinding OpenPaneLength}"
                                 PaneBackground="{ThemeResource NavigationViewDefaultPaneBackground}"
                                 Grid.Row="1"
-                                contract7Present:CornerRadius="{ThemeResource OverlayCornerRadius}">
+                                contract7Present:CornerRadius="{ThemeResource NavigationViewPaneContentGridCornerRadius}">
```

Comment:
we might have a converter that can filter out the values you need instead of creating a new resource:)